x-pack/plugins/infra/public/pages/metrics/metric_detail/index.tsx

```diff
@@ -19,11 +21,25 @@ export const NodeDetail = () => {
     params: { type: nodeType, node: nodeName },
   } = useRouteMatch<{ type: InventoryItemType; node: string }>();
 
-  useMetricsBreadcrumbs([
-    {
-      text: nodeName,
-    },
-  ]);
+  const breadCrumbs: ChromeBreadcrumb[] = [];
+
+  const hostsLinkProps = useLinkProps({
+    app: 'metrics',
+    pathname: 'hosts',
+  });
+
+  if (nodeType === 'host') {
+    breadCrumbs.push({
+      ...hostsLinkProps,
+      text: 'Hosts',
```

Comment:
You missed the translation in this text. You can find some examples in the code on how to use the `i18n`